### PR TITLE
Bxmsdoc 1914 master

### PR DESCRIPTION
**Pull request by mramendi**
JIRA https://issues.jboss.org/browse/BXMSDOC-1914

rendered document http://file.ork.redhat.com/~mramendi/BXMSDOC-1914/#guided_decision_tables_con

### CODE REVIEW

File: docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc [hunk 1]
```diff
@@ -7,21 +7,22 @@ You can use guided decision tables to define rule attributes, meta-data, conditi
 . Go to *Menu* -> *Design* -> *Projects* and select your project.
 . Click *Create New Asset* -> *Guided Decision Table*.
 . Enter an informative *Guided Decision Table* name and select the appropriate *Package*.
-. Select *Use Wizard* to finish setting up the table in the Wizard, or leave this option unselected to finish creating the table and specify remaining configurations in the guided decision table editor.
-+
-The Wizard helps you define your imports, facts, patterns, and columns, but not the rows. Rows are added in the guided decision table editor after initial table setup.
```
Comment (emmurphy1) [inline docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc]: Why delete this sentance?

File: docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc [hunk 2]
```diff
@@ -7,21 +7,22 @@ You can use guided decision tables to define rule attributes, meta-data, conditi
 . Go to *Menu* -> *Design* -> *Projects* and select your project.
 . Click *Create New Asset* -> *Guided Decision Table*.
 . Enter an informative *Guided Decision Table* name and select the appropriate *Package*.
-. Select *Use Wizard* to finish setting up the table in the Wizard, or leave this option unselected to finish creating the table and specify remaining configurations in the guided decision table editor.
```
Comment (emmurphy1) [inline docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc]: ..in the Wizard.. w should be l/c

File: docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc [hunk 3]
```diff
@@ -7,21 +7,22 @@ You can use guided decision tables to define rule attributes, meta-data, conditi
...
-+
-The Wizard helps you define your imports, facts, patterns, and columns, but not the rows. Rows are added in the guided decision table editor after initial table setup.
-+
+. Select *Use Wizard* to finish setting up the table in the Wizard, or leave this option unselected to finish creating the table and specify remaining configurations in the guided decision table editor.
```
Comment (emmurphy1) [inline docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc]: in the Wizard.. w should be l/c. I know this isn't new text, but while you are editing...

File: docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc (hunk 1 above)
Comment (mramendi) [reply docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc]: Moved, not deleted. The doc covers the wizard a bit later, where there is a screenshot of the wizard. Covering the wizard here, next to a different screenshots, was in my view confusing, so I moved the information lower, slightly rephrasing it.

File: docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc (hunk 3 above)
Comment (mramendi) [reply docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc]: I'll fix that but need to understand how to work pull requests if I edit stuff. If I rescind this pull request we lose the discussion?

**Pull request merged**